### PR TITLE
Break getPatchesForPodResourceRequest into smaller functions

### PR DESCRIPTION
**Pull request by jbartosik**
Prepare for setting limits

### CODE REVIEW

File: vertical-pod-autoscaler/pkg/admission-controller/logic/server.go [hunk 1]
```diff
@@ -125,6 +96,53 @@ func (s *AdmissionServer) getPatchesForPodResourceRequest(raw []byte, namespace
...
+
+func getAddResourcesSubPatch(i int, kind string) patchRecord {
+	return patchRecord{
+		Op:    "add",
+		Path:  fmt.Sprintf("/spec/containers/%d/resources/%s", i, kind),
+		Value: v1.ResourceList{},
+	}
+}
+
+func getAddResourceParamPatch(i int, kind, resource v1.ResourceName, quantity resource.Quantity) patchRecord {
```
Comment (bskiba) [inline vertical-pod-autoscaler/pkg/admission-controller/logic/server.go]: nit: Shouldn't kind be explicitly a string?

File: vertical-pod-autoscaler/pkg/admission-controller/logic/server.go [hunk 2]
```diff
@@ -125,6 +96,53 @@ func (s *AdmissionServer) getPatchesForPodResourceRequest(raw []byte, namespace
 	return patches, nil
 }
 
+func getAddResourcesPatches(i int) patchRecord {
```
Comment (bskiba) [inline vertical-pod-autoscaler/pkg/admission-controller/logic/server.go]: getAddResourcesPatches is initializes resource requirements to an empty object, so maybe put "empty" explicitly in the function name. Same for getAddResourcesSubpatch. Then the comments in getContainerPatch would become unneeded.

File: vertical-pod-autoscaler/pkg/admission-controller/logic/server.go (hunk 1 above)
Comment (bskiba) [inline vertical-pod-autoscaler/pkg/admission-controller/logic/server.go]: getAddResourceRequirementValuePatch?

File: vertical-pod-autoscaler/pkg/admission-controller/logic/server.go (hunk 2 above)
Comment (jbartosik) [reply vertical-pod-autoscaler/pkg/admission-controller/logic/server.go]: Done.

File: vertical-pod-autoscaler/pkg/admission-controller/logic/server.go (hunk 1 above)
Comment (jbartosik) [reply vertical-pod-autoscaler/pkg/admission-controller/logic/server.go]: Done.

Comment (jbartosik) [reply vertical-pod-autoscaler/pkg/admission-controller/logic/server.go]: Done.

**Pull request merged**